src/index.ts

```diff
@@ -338,7 +342,7 @@ class MayanRouteBase<N extends Network>
             quote.details!,
             originAddress,
             destinationAddress,
-            this.referrerAddress(),
+            this.referrerAddress(request.fromChain.chain),
```

Comment:
Does the swap get bps info from the quote itself? Since its not being passed in this api?

Reply:
@mohandesjiri qq, you mention it selects the source chain, but I think I saw the referrer fee get deposited to the address for destination chain.

Reply:
We are wondering if the referrer fee always gets paid out on the destination chain